insteon_mqtt/device/Thermostat.py

```diff
@@ -181,50 +181,26 @@ def pair(self, on_done=None):
         seq.run()
 
     #-----------------------------------------------------------------------
-    def refresh(self, force=False, on_done=None):
-        """Refresh the current device state and database if needed.
+    def addRefreshData(self, seq, force=False):
+        """Add commands to refresh any internal data required.
 
-        This sends a ping to the device.  The reply has the current device
-        state (on/off, level, etc) and the current db delta value which is
-        checked against the current db value.  If the current db is out of
-        date, it will trigger a download of the database.
+        The base class uses this update the device catalog ID's and firmware
+        if we don't know what they are.
 
-        This will send out an updated signal for the current device status
-        whenever possible.
+        This is split out of refresh() so derived classes that override
+        refresh can also get this information.
 
-        In addition, this also runs the 'get_status' command as well, which
-        asks the thermostat for the current state of its attributes as well
-        the current units selected on the device.  If you are seeing errors
-        in temperatures that look like C and F are reversed, running a refresh
-        may fix the issue.
+        The base refresh only checks the DB value for a thermostat, all other
+        states are determined by get_status()
 
         Args:
+          seq (CommandSeq): The command sequence to add the command to.
           force (bool):  If true, will force a refresh of the device database
                 even if the delta value matches as well as a re-query of the
                 device model information even if it is already known.
-          on_done: Finished callback.  This is called when the command has
-                   completed.  Signature is: on_done(success, msg, data)
         """
-        LOG.info("Device %s cmd: fan status refresh", self.addr)
-
-        seq = CommandSeq(self, "Refresh complete", on_done, name="DevRefresh")
-
-        # Send a 0x19 0x03 command to get the fan speed level.  This sends a
-        # refresh ping which will respond w/ the fan level and current
-        # database delta field.  Pass skip_db here - we'll let the dimmer
-        # refresh handler above take care of getting the database updated.
-        # Otherwise this handler and the one created in the dimmer refresh
-        # would download the database twice.
-        msg = Msg.OutStandard.direct(self.addr, 0x19, 0x03)
```

Comment:
I don't remember why I had written cmd2=0x03.  It may be that we used to do things differently.  Now, the generic refresh command only gets the aldb delta.  The device status is handled by the `get_status` requested which is added to the queue via `addRefreshData()`

Reply:
As far as I can tell, the old code was copied from device/Fanlinc.py.  Fanlinc still uses cmd2=0x03 and had the same block comments prior to this refactoring.  It seems reasonable to me that the 0x03 value wouldn't necessarily apply to the thermostat, too, so the changes here seem fine to me.